api/Navigator.json

```diff
@@ -992,6 +992,83 @@
           }
         }
       },
+      "maxTouchPoints": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/Navigator/maxTouchPoints",
+          "support": {
+            "chrome": {
+              "version_added": "35"
+            },
+            "chrome_android": {
+              "version_added": "35"
+            },
+            "edge": {
+              "version_added": null
+            },
+            "edge_mobile": {
+              "version_added": null
+            },
+            "firefox": [
+              {
+                "version_added": "59"
+              },
+              {
+                "version_added": "29",
+                "flags": [
+                  {
+                    "type": "preference",
+                    "name": "dom.w3c_pointer_events.enabled",
+                    "value_to_set": "true"
+                  }
+                ]
+              }
+            ],
+            "firefox_android": {
+              "version_added": "29",
+              "flags": [
+                {
+                  "type": "preference",
+                  "name": "dom.w3c_pointer_events.enabled",
+                  "value_to_set": "true"
+                }
+              ],
+              "notes": "Not enabled by default yet, see <a href='https://bugzil.la/1426786'>bug 1426786</a>."
```

Comment:
Hmm, but then in the mean time there's no good way for users to know why this is disabled on Firefox for Android. When I saw this initially I thought the data was wrong, I think the bugzilla bug is important to include :)

Reply:
I definitely agree that the bug link should remain. I just thought the data itself showed that this was "Not enabled by default yet" so that part of the note text would be redundant. Have I misunderstood something?

Reply:
Ah, no that's fair. I'll change it.